sdk/go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/armon/go-metrics v0.4.1
 	github.com/armon/go-radix v1.0.0
 	github.com/cenkalti/backoff/v3 v3.2.2
-	github.com/docker/docker v24.0.5+incompatible
+	github.com/docker/docker v24.0.7+incompatible
```

Comment:
The CI setup is failing due to a missing go.sum entry for this dependency. You can resolve with the following:

```
	github.com/docker/docker@v24.0.7+incompatible: missing go.sum entry for go.mod file; to add it:
	go mod download github.com/docker/docker
```